lib/web_ui/test/canvaskit/embedded_views_test.dart

```diff
@@ -284,14 +286,15 @@ void testMain() {
 
     test('renders overlays on top of platform views', () async {
       expect(SurfaceFactory.instance.debugCacheSize, 0);
+      expect(configuration.canvasKitMaximumSurfaces, 8);
       final CkPicture testPicture =
           paintPicture(const ui.Rect.fromLTRB(0, 0, 10, 10), (CkCanvas canvas) {
         canvas.drawCircle(const ui.Offset(5, 5), 5, CkPaint());
       });
 
       // Initialize all platform views to be used in the test.
       final List<int> platformViewIds = <int>[];
-      for (int i = 0; i < configuration.canvasKitMaximumSurfaces * 2; i++) {
+      for (int i = 0; i < 16; i++) {
```

Comment:
Why not leave this as `config.maximumSurfaces * 2`? Wouldn't we want to test with 32 surfaces if we bumped canvaskitMaximumSurfaces to 16, for example?

(I know that the test would fail, but the relationship of this `16` with canvaskitMaximumSurfaces is tenuous to me.)

Reply:
I did it because the "expectSceneMatches" check already has an explicit number of platform views in it.